cmd/acraserver/acraserver.go

```diff
@@ -152,7 +158,13 @@ func main() {
 		}
 	}
 
-	server, err := NewServer(config, keyStore)
+	var server *SServer
+	if os.Getenv("_GRACEFUL_RESTART") == "true" {
+		server, err = NewFromFD(config, keyStore, 3, 4)
```

Comment:
what means `3, 4`? Can we use here some constants? 
plus why not extend NewServer and add new function?

Reply:
Constants used